proposals/static-lambdas.md

```diff
@@ -0,0 +1,32 @@
+# Static lambdas
+
+## Summary
+
+Support lambdas that disallow capturing state from the enclosing scope.
+
+## Motivation
+
+Avoid unintentionally capturing state from the enclosing context.
+
+## Detailed design
+
+A lambdas with `static` cannot capture state from the enclosing scope.
+As a result, locals, parameters, and `this` from the enclosing scope are not available within a `static` lambda.
+
+A `static` lambda cannot reference instance members from an implicit or explicit `this` or `base` reference.
```

Comment:
I suspect you intend that such a reference is a compile-time error.  Please say that, and identify the sections of the ECMA specification that require modifications.